src/ngMessageFormat/messageFormatCommon.js

```diff
@@ -5,11 +5,11 @@
 // This file is compiled with Closure compiler's ADVANCED_OPTIMIZATIONS flag! Be wary of using
 // constructs incompatible with that mode.
 
-var $interpolateMinErr = angular['$interpolateMinErr'];
+var $interpolateMinErr = window['angular']['$interpolateMinErr'];
```

Comment:
This shouldn't be necessary, since _normally_ the code would be wrapped into an IIFE and passed `window.angular` as an argument:

```js
(function (window, angular, undefined) { ... })(window, window.angular);
```

Any idea why it doesn't work ?